specification/reservations/resource-manager/Microsoft.Capacity/stable/2022-03-01/reservations.json

```diff
@@ -572,6 +572,80 @@
         }
       }
     },
+    "/providers/Microsoft.Capacity/reservationOrders/{reservationOrderId}/reservations/{reservationId}/archive": {
```

Comment:
APis look fine , but you would need to add a new api version since adding apis to stable version is not allowed